solr/core/src/java/org/apache/solr/servlet/ResponseUtils.java

```diff
@@ -42,6 +43,26 @@ private ResponseUtils() {}
    * @see #getTypedErrorInfo(Throwable, Logger)
    */
   public static int getErrorInfo(Throwable ex, NamedList<Object> info, Logger log) {
+    return getErrorInfo(ex, info, log, null);
+  }
+
+  /**
+   * Adds the given Throwable's message to the given NamedList.
+   *
+   * <p>Primarily used by v1 code; v2 endpoints or dispatch code should call {@link
+   * #getTypedErrorInfo(Throwable, Logger)}
+   *
+   * <p>If the response code is not a regular code, the Throwable's stack trace is both logged and
+   * added to the given NamedList.
+   *
+   * <p>Status codes less than 100 are adjusted to be 500.
+   *
+   * <p>Stack trace will not be output if hideTrace=true OR system property
+   * solr.hideStackTrace=true.
+   *
+   * @see #getTypedErrorInfo(Throwable, Logger)
+   */
+  public static int getErrorInfo(Throwable ex, NamedList<Object> info, Logger log, SolrCore core) {
```

Comment:
could you replace 'core' instance with just the hideStackTrace flag? there is no reason to pass in the entire core.

Reply:
Done in latest commit.  Thanks.